chromium_src/chrome/browser/ui/webui/settings/settings_localized_strings_provider.cc

```diff
@@ -303,6 +303,9 @@ void BraveAddCommonStrings(content::WebUIDataSource* html_source,
     {"ipfsRotateKeyDialogTitle", IDS_SETTINGS_IPFS_ROTATE_KEY_DIALOG_TITLE},
     {"ipfsRotationLaunchError", IDS_SETTINGS_IPFS_ROTATION_LAUNCH_ERROR},
     {"ipfsRotationStopError", IDS_SETTINGS_IPFS_ROTATION_STOP_ERROR},
+    {"ipfsExportKey", IDS_SETTINGS_IPNS_KEY_EXPORT_ITEM},
+    {"ipfsKeyRemove", IDS_SETTINGS_IPNS_KEY_REMOVE_ITEM},
+    {"ipfsExportKeysError", IDS_SETTINGS_IPNS_KEYS_EXPORT_ERROR},
```

Comment:
Nit: maybe keep the map key names consistent with the IDS names, e.g. `IDS_SETTINGS_IPNS_KEY_EXPORT_ITEM` -> `ipnsKeyExport`, etc.

Reply:
will fix in next pr thanks